src/util/thousandSeparation.ts

```diff
@@ -1,2 +1,4 @@
-export const thousandSeparation = (number: number): string =>
-	number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ".");
+export const thousandSeparation = (number?: number): string =>
+	number
+		? number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ".")
```

Comment:
Vet du at formattering blir riktig når for alle "vanlige" tall 1, 10, 100, 1000, 10000, 100000 osv. Trenger du test her?

Reply:
> Vet du at formattering blir riktig når for alle "vanlige" tall 1, 10, 100, 1000, 10000, 100000 osv. Trenger du test her?

Har tests på vanlige integers, og det virker greit. Men den breaker på floating tall hvis der er flere end 3 decimaler. Men det burde der aldrig komme på korrekt data. Kritisk å fikse tror du?

Reply:
Begge sider burde nok kutte på 2

Reply:
Må man bruke regex her?